src/fsharp/PackageManager.Integration.fs

```diff
@@ -38,15 +38,17 @@ type PackageManager = {
     ToolName: string
     Name:string }
 
-let RegisteredPackageManagers : PackageManager list = 
+let RegisteredPackageManagers = lazy (
```

Comment:
But Lazy.Force (what a name!) won't reevaluate right? say in script A I register some handler, and in script B I load script A and rely on those handlers, will this work?